render/domFor.js

```diff
@@ -1,6 +1,6 @@
 "use strict"
 
-var delayedRemoval = new WeakMap
+var delayedRemoval = require("../render/delayedRemoval")
```

Comment:
Let's not leave redundant import path components here.

```suggestion
var delayedRemoval = require("./delayedRemoval")
```

Reply:
fixed